recipe/meta.yaml

```
@@ -10,12 +10,13 @@ source:
   sha256: 093db6840caad909ddb5159ba0a64a4822155e0bc18d2cbabc0c8a5bcf351026
   patches:
     - 0001-Fix-for-type-of-index-on-llvm-compiler.patch
+    - pp-sketchlib-aarch64.patch  # [aarch64]
 
 build:
-  number: 3
+  number: 4
   skip: true  # [py<38 or win or cuda_compiler_version in ("9.2", "10.0", "10.1", "10.2", "11.0")]
   script:
```

Would start with something like this for setting `CUDA_HOME`. We may potentially want to add `PREFIX` too, but let's skip it for now and see if this works without it

```suggestion
  script_env:
    # Ensure that CUDA includes can be found by the host compiler
    - CUDA_HOME={{ os.sep.join([BUILD_PREFIX, "targets", "x86_64-linux"]) }}  # [linux64 and cuda_compiler_version not in ("None", "11.8")]
    - CUDA_HOME={{ os.sep.join([BUILD_PREFIX, "targets", "sbsa-linux"]) }}    # [linux and aarch64 and cuda_compiler_version not in ("None", "11.8")]
    - CUDA_HOME={{ os.sep.join([BUILD_PREFIX, "targets", "ppc64le-linux"]) }}    # [linux and ppc64le and cuda_compiler_version not in ("None", "11.8")]
    - SKETCHLIB_INSTALL=conda                                                 # [linux and cuda_compiler_version not in ("None", "11.8")]
  script:
```